lib/components/user/monitored-trip/saved-trip-list.tsx

```diff
@@ -96,63 +110,58 @@ class TripListItem extends Component<ItemProps, ItemState> {
     // - get the 'to' location of the last leg.
     const from = legs[0].from
     const to = legs[legs.length - 1].to
+    const editTripPath = `${TRIPS_PATH}/${trip.id}`
+    const { LegIcon } = this.context
     return (
       <Panel>
         <TripPanelHeading>
-          {shouldRenderAlerts && (
-            <TripPanelAlert onClick={this._handleEditTrip}>
-              <IconWithText Icon={TriangleExclamation}>
-                <FormattedMessage
-                  id="components.SavedTripList.alertTag"
-                  values={{ alert: alerts.length }}
-                />
-              </IconWithText>
-            </TripPanelAlert>
-          )}
-          <Panel.Title>
-            <TripHeader>{trip.tripName}</TripHeader>
-            <small>
-              <FormattedMessage
-                id="components.SavedTripList.fromTo"
-                values={{
-                  from: <strong>{from.name}</strong>,
-                  to: <strong>{to.name}</strong>
-                }}
-              />
-            </small>
-          </Panel.Title>
+          <TripPanelTitle>
+            <Panel.Title>
+              <TripHeader>{trip.tripName}</TripHeader>
+            </Panel.Title>
+            <Link
+              title={intl.formatMessage({
+                id: 'components.SavedTripEditor.editSavedTrip'
+              })}
+              to={editTripPath}
+            >
+              <Edit height={18} />
+              <InvisibleA11yLabel>
+                <FormattedMessage id="components.SavedTripEditor.editSavedTrip" />
+              </InvisibleA11yLabel>
+            </Link>
+          </TripPanelTitle>
+          <RouteBlockGrid>
+            <MetroItineraryRoutes
```

Comment:
This is having some problems with custom route renderers. I'm a little confused as to why. I think if we add a TODO for now we should be ok for now. Eager to get this merged